SemanticUI/Modules/Common.elm

```diff
@@ -0,0 +1,36 @@
+module SemanticUI.Modules.Common exposing (Render, WrappedContent, WrappedNode)
+
+import Html exposing (..)
+
+
+{-| Convenience alias for a HTML DOM Node render function
+-}
+type alias Render msg =
+    List (Attribute msg) -> List (Html msg) -> Html msg
```

Comment:
It feels to me like this should be `Renderer` or something like that, similar to `Builder`/`Parser`/`Decoder` etc...